README.md

```diff
@@ -1,11 +1,11 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-
+# Capstone Project
```

Comment:
# Optional:
- Please add lines describing what is the project about. Here will be the first page "a recruiter " will read to understand your project and it can help you to land a first Job contract